content/blog/2023/08/28/2023-08-28-number-of-open-issues-probe.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Number of Open Issues Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/number-of-open-issues-probe.png
+sig:
+- gsoc
+---
+
+== Introduction
+
+An alarming number of issues in a plugin may signify that the plugin is outdated, in need of a new maintainer, or in need of a complete revamp.
+Open issues might also mean that there are a lot of new feature requests, which in turn means the plugin has quite a number of active users.
+The number of open issue probe counts the number of open issues in the issue tackers.
+
+NOTE: As of now, the probe only counts open issues in GitHub and the JIRA issue tacker.
+
+== Importance of the probe
+
+The goal of the probe is to help maintainers and contributors identify plugins they want to invest their time into.
+This would lead to maintainers keeping plugins updated and useful for the users.
```

Comment:
Not sure about this sentence TBH.

This probe will lead to maintainers keeping their issue trackers up-to-date or addressing issues, not keeping plugins updated?